guides/int8_quantization_in_keras.py

```diff
@@ -0,0 +1,210 @@
+"""
+Title: 8-bit Integer Quantization in Keras
+Author: [Jyotinder Singh](https://x.com/Jyotinder_Singh)
+Date created: 2025/10/14
+Last modified: 2025/10/14
+Description: Complete guide to using INT8 quantization in Keras and KerasHub
```

Comment:
In some places it's INT8, other places it's int8, and some other places `int8` with backquotes. Let's standardize.